readthedocs/projects/fixtures/test_data.json

```diff
@@ -1,346 +1,1161 @@
 [
```

Comment:
The JSON file is not _maintained_, it's an automated artifact. I will respond with your own instructions:

https://github.com/readthedocs/readthedocs.org/issues/8914#issuecomment-1046916832

I wouldn't say that _I_ upgraded the whole file, I re-used the work of a contributor who had already done it, then I verified that it works which it did :+1: 

My immediate reaction: Creating a script like this is moving the goal line for the second time: Initially, I wanted to change `master` to `main` on the readthedocs.org fixture, I added `test-builds` while doing that -- you suggested to fix another issue and I did include the work already submitted as a PR by another contributor. With a suggestion to create a utility script, I think we are too far off the radar for what this PR should do.

I think that the work here is good to go:

1) It fixes an immediate bug
2) It's an overall improvement of previous situation
3) It's possible to defer a utility script to a separate issue.

Reply:
Yeah. I'm not saying the creation of the script like this has to be done in this PR. It's a separate issue 👍

However, I'd like to do that eventually because even if the JSON is generated automatically by Django, we have to update it from a fresh DB. Otherwise, a lot of extra data is added. In the end, it's harder to maintain than just a Python file with exactly what we want.